models/sessions.js

```diff
@@ -0,0 +1,101 @@
+'use strict';
+
+var Db        = require('../config/database');
+var mongoose  = require('mongoose');
+var crypto    = require('crypto');
+
+// Generates a random string to use as a session token
+function generateToken() {
+  return crypto.randomBytes(256).toString('base64');
+}
+
+// Create the schema for a session
+var SessionSchema = mongoose.Schema({
+
+  // Defaults to a string of 256 characters
+  token     : { type : String, default : generateToken },
+
+  // Defaults to the current time and expires in 7 days
+  createdAt : { type : Date, default : Date.now, expires : '7d' },
+
+  // Defaults to the current time + 7 days
+  expireAt  : { type : Date, default : Date(Date.now + 604800000) },
```

Comment:
This should be `expireAt  : { type : Date, default : Date.now + 604800000 },` instead